runtime/bootstrapper.ts

```diff
@@ -1,29 +1,32 @@
 import { BBAuth, BBOmnibar } from '@blackbaud/auth-client';
+import { BBHelp } from '@blackbaud/help-client';
 
 import { SkyAppBootstrapConfig } from './bootstrap-config';
 
 export class SkyAppBootstrapper {
   public static bootstrapConfig: SkyAppBootstrapConfig;
 
   public static processBootstrapConfig(): Promise<any> {
+    let promises: Promise<any>[] = [];
+
     if (SkyAppBootstrapper.bootstrapConfig) {
-      let authPromise: Promise<any>;
 
+      // BBAuth init
       if (SkyAppBootstrapper.bootstrapConfig.auth) {
-        authPromise = BBAuth.getToken();
-      } else {
-        authPromise = Promise.resolve();
+        promises.push(BBAuth.getToken());
       }
 
+      // BBOmnibar init
       if (SkyAppBootstrapper.bootstrapConfig.omnibar) {
-        return authPromise.then(() => {
-          BBOmnibar.load(SkyAppBootstrapper.bootstrapConfig.omnibar);
-        });
-      } else {
-        return authPromise;
+        promises.push(BBOmnibar.load(SkyAppBootstrapper.bootstrapConfig.omnibar));
+      }
+
+      // BBHelp init
+      if (SkyAppBootstrapper.bootstrapConfig.help) {
+        promises.push(BBHelp.load(SkyAppBootstrapper.bootstrapConfig.help));
       }
     }
 
-    return Promise.resolve();
+    return Promise.all(promises);
```

Comment:
I started to ask if this would resolve in the case of auth, omnibar, and help being set as false, which means `bootstrapConfig` is never set.  

Looking at https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Promise/all says "If an empty array is passed, then this method resolves immediately.", so question answered.